new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.styles.js

```diff
@@ -21,5 +23,14 @@ export default {
   },
   col2: {
     width: 136
+  },
+  cashbox: {
+    '& p': {
+      marginRight: 4,
+      fontSize: 20
+    }
+  },
```

Comment:
isn't this something we can do straight on the cashbox component?

Reply:
This would make the cashbox element on other pages (CashCassettes) have the larger font size by default, which is not intended. Either the notifications page or the cash cassettes page would need to override the styling for it to work as intended on both pages